setup.py

```diff
@@ -59,7 +59,7 @@
         # 'jupyterlab-git==0.21.1',
         'kfp-notebook>=0.14.0,<0.15.0',
         'kfp==1.1.0',
-        'minio>=5.0.7',
+        'minio==6.0.2',
```

Comment:
I think rather than hard setting this we should set a range that allows update to future 6.X patches